src/Bridge/Symfony/Bundle/Resources/config/swagger.xml

```diff
@@ -30,7 +30,7 @@
         <!-- Action -->
 
         <service id="api_platform.swagger.action.ui" class="ApiPlatform\Core\Bridge\Symfony\Bundle\Action\SwaggerUiAction">
-            <argument type="service" id="twig" on-invalid="ignore" />
+            <argument type="service" id="twig" />
```

Comment:
`SwaggerUiAction` does not work without Twig (obviously). `ApiPlatformBundle` should have a dependency on `symfony/twig-bundle`, but of course currently we don't have a separate composer.json for the bundle (something we definitely should have).

p/s: Yes, of course the rest of the Swagger integration is still functional without Twig, but it's the bundle we're talking about. And yes, of course, if we want to be pedantic, the bundle does not need Twig if we don't `enable_swagger`. But considering that this line is in `swagger.xml`, that's clearly not the case.